chingsley/ireporter
# perform general testing and debugging

#### What does this PR do?
This pull request deals with general testing and debugging of the projet

#### Description of Task to be completed?
- go to chingsley.github.io/gh-pages/
- Click on sign up and register
- Then click on sign in and start testing the app.

#### How should this be manually tested?
- clone the repo
- cd into the root directory
- run `npm install' to install the necessary dependencies
- run `npm test' to see the tests pass
- run `npm start' to start the server
- then cd into challenge-4/UI and run the command `live-server`. This opens the front end part of the app for easy testing and debugging.
